mcm/tools/locator.py

```diff
@@ -60,4 +143,75 @@ def database_credentials(self):
         cred_header = os.getenv("COUCH_CRED", "")
         if not cred_header:
             raise RuntimeError("Set COUCH_CRED to the CouchDB authentication header")
-        return cred_header
\ No newline at end of file
+        return cred_header
+
+    def stats_database_url(self):
+        """
+        Retrieve the Stats2 CouchDB connection URL.
+        This can be overwritten using the environment variable:
+            `MCM_STATS2_DB_URL`
+
+        Returns:
+            str: Stats2 CouchDB connection URL.
+        """
+        custom_url = os.getenv("MCM_STATS2_DB_URL")
+        default_url = "http://vocms074.cern.ch:5984/"
+        if custom_url:
+            self.logger.info("Stats2 custom connection URL: %s", custom_url)
+            return custom_url
+
+        return default_url
+
+    def mcm_executor_node(self):
+        """
+        Returns the remote hostname used by McM to
+        perform SSH executions for processing some tasks
+        like approving requests, upload request configuration to ReqMgr2,
+        or execute injection commands.
+
+        This can be overwritten using the environment variable:
+            `MCM_EXECUTOR_HOST`
+
+        Returns:
+            str: McM executor node hostname.
+        """
+        custom_node = os.getenv("MCM_EXECUTOR_HOST")
+        default_node = "vocms0481.cern.ch"
+        if custom_node:
+            self.logger.info("Using a custom McM executor node: %s", custom_node)
+            return custom_node
+        return default_node
+
+    def host(self):
+        """
+        Returns the binded host for the web application.
+        """
+        return os.getenv("MCM_HOST", "0.0.0.0")
+
+    def port(self):
+        """
+        Returns the port for the web application.
+        """
+        return os.getenv("MCM_PORT", "8000")
+
+    def debug(self):
+        """
+        Enables the DEBUG level for logging.
+        """
+        return bool(os.getenv("MCM_DEBUG"))
+
+    def logs_folder(self):
+        """
+        Retrieve the absolute path for the log folder.
+        This can be overwritten using the environment variable:
+            `MCM_LOG_FOLDER`
+
+        Returns:
+            str: Log folder absolute path.
+        """
+        custom_path = os.getenv("MCM_LOG_FOLDER")
+        default_path = os.path.join(os.getcwd(), "logs")
+        if custom_path:
+            self.logger.info("Using a custom log folder: %s", custom_path)
+            return custom_path
+        return default_path
```

Comment:
Only compute `default_path` here if needed

Reply:
I think this is more style. I don't see any issue with the current behavior. Closing this comment.

Reply:
`os.env` is a syscall, which is relatively expensive

Reply:
Okay, given that context, I am going to perform the change